modules/custom/cu_core/cu_core.install

```diff
@@ -709,6 +709,75 @@ function cu_core_update_7024() {
 }
 
 /**
+ * Add github forms bundle or remove bundle if not in use.
+ */
+function cu_core_update_7025() {
+  // We need to know the 'statistics_id'.
+  module_load_include('inc', 'atlas_statistics');
+  $atlas_statistics = atlas_statistics_get_statistics();
+
+  $result = $atlas_statistics['nodes_by_type']['webform'];
+
+  if ($result < 1 || $result == null) {
+    // disable and uninstall modules
+    if (module_exists('captcha')) {
```

Comment:
can these modules be disabled and uninstalled in one step? Loop through check of modules and then disable after creating the list of modules. This might help with the fatal errors on updb?